tutorials/classify-maizeland-ng/index.md

```diff
@@ -0,0 +1,235 @@
+c. Test Classification outputs, generate validation matrix, and calculate accuracy
+
+3.	Calculate area and Export Outputs
+
+4.	Final notes
+
+
+## 1. Ingesting, Importing, and Visualizing Data
+
+a. Ingest the Nigerian boundary as the focal geography and maize target region boundary as the area of interest (AOI). Using the code below, you will import a FeatureCollection object, and filter by "Country" to select "Nigeria". FeatureCollections are groups of features (spatial data and attributes). Filter is the method to extract a specific set of features from a feature collection. Assign the output to a variable called "nigerianBorder". The imagery analyses will be limited to the maize target region in Nigeria, i.e. the region that accounts for ~70% of Nigeria's maize production. Therefore, you will import a predefined shapefile layer (already converted to GEE asset) and assign to the variable "aoi". The maize target region asset can be assessed here (https://code.earthengine.google.com/?asset=users/juliusadewopo/MzeTargetRegion_alt_dslv2).  Display both layers to the map it using Map.addLayer() and tweak the symbology with the color parameters specified below.
+
+```js
+// Ingest country boundaries feature collection.
+var dataset = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017');
+
+// Apply filter where country name equals Nigeria.
+var nigeriaBorder = dataset.filter(ee.Filter.eq('country_na', 'Nigeria'));
+
+// Print new "nigeriaBorder" object and explore features and properties.
+// There should only be one feature representing Nigeria.
+print(nigeriaBorder);
+
+// Center the map view on Nigeria, otherwise, the view defaults to the Country of your IP address
+Map.centerObject(nigeriaBorder, 6);
+
+//Setting color parameters for Nigeria Boundary
+var shown = true; // true or false, 1 or 0 
+var opacity = 0.2; // number [0-1]
+var nameLayer = 'map'; // string
+var visParams = {color: 'red'}; // dictionary: 
+
+//Add Nigeria as a layer to the map view
+Map.addLayer(nigeriaBorder, visParams, nameLayer, shown, opacity);
+
+//Setting color parameters for Maize target area boundary
+var shown = true; // true or false, 1 or 0 
+var opacity = 0.5; // number [0-1]
+var nameLayer = 'map2'; // string
+var visParams = {color: 'brown', strokeWidth: 5}; // dictionary:
+Map.addLayer(aoi, visParams, nameLayer, shown, opacity);
+```
+
+b. Ingest groundtruth data for georeferenced locations where maize (and other crops) were cultivated during the growing season of 2017 (June - Oct). The data has been pre-processed and randomly split (70:30) into training and testing datasets. Import the training dataset as GEE asset here (https://code.earthengine.google.com/?asset=users/juliusadewopo/Mzetrgt_Train17_Rev), and the testing dataset here (https://code.earthengine.google.com/?asset=users/juliusadewopo/Mzetrgt_Test17_Rev). Assing variable names "trainpts" and "testpts" to the training and testing points, respectively, and add them as layer to the map view.
```

Comment:
groundtruth and ground-truth => ground truth throughout.

Reply:
done